utils/train.py

```diff
@@ -110,12 +111,12 @@ def train(config):
     dev_loader = data.DataLoader(
         dev_set,
         batch_size=min(len(dev_set), 16),
-        shuffle=True,
+        shuffle=False,
         collate_fn=dev_set.collate_fn)
     test_loader = data.DataLoader(
         test_set,
-        batch_size=min(len(test_set), 16),
-        shuffle=True,
+        batch_size=len(test_set),
```

Comment:
Can we stick to `min(len(test_set), 16)`? The entire test set isn't guaranteed to fit in the VRAM, for different choices of `test_pct`.

Reply:
Of course it does not affect the performance